src/types.ts

```diff
@@ -439,12 +434,18 @@ export interface GetSwapPriceResponse extends BasePriceResponse {}
 
 export type GetTokenPricesResponse = Price[];
 
-export interface GetMetaTransactionQuoteResponse extends BasePriceResponse {
+export interface GetMetaTransactionQuoteResponseV0 extends BasePriceResponse {
     zeroExTransactionHash: string;
     zeroExTransaction: ZeroExTransaction;
     orders: SignedOrder[];
 }
 
+export interface GetMetaTransactionQuoteResponseV1 extends BasePriceResponse {
+    mtxHash: string;
```

Comment:
thoughts on just keeping this inline with the 0x Exchange version? I don't thhink the user needs to know there is a difference?

Reply:
I messed up the type on `mtx`. It should just be `ExchangeProxyMetaTransaction`. Overlooking that, I made separate interfaces because it felt weird also calling EP mtxs `zeroExTransaction`s since the `ZeroExTransaction` naming has always been associated with Exchange mtxs.